Description: Always use FFmpeg and do not set include directory
 Fixes the search path for FFmpeg and drops the manual /usr/include addition to
 the include path which breaks C++ STL headers with GCC-6+.
Author: James Cowgill <jcowgill@debian.org>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/ffDiaporama/ffDiaporama.pro
+++ b/src/ffDiaporama/ffDiaporama.pro
@@ -85,35 +85,9 @@ unix {
             message("Linux x86 build")
         }
 
-        exists(/opt/ffmpeg/include/libswresample/swresample.h) {         #------ conditionnaly includes from Sam Rog packages for Ubuntu
-            message("Use SAM ROG Packages from /opt/ffmpeg")
-            INCLUDEPATH += /opt/ffmpeg/include/
-            LIBS        += -L"/opt/ffmpeg/lib"
-            DEFINES += USELIBSWRESAMPLE
-            LIBS    += -lswresample                                             #------ conditionnaly include libswresample
-        } else:exists(/usr/include/ffmpeg/libswresample/swresample.h) {         #------ Specific for Fedora
-            message("Use ffmpeg in /usr/include/ffmpeg")
-            DEFINES += USELIBSWRESAMPLE
-            INCLUDEPATH += /usr/include/ffmpeg/
-            LIBS    += -lswresample                                             #------ conditionnaly include libswresample
-        } else:exists(/usr/include/libswresample/swresample.h) {                #------ Specific for openSUSE
-            message("Use ffmpeg in /usr/include")
-            INCLUDEPATH += /usr/include/
-            DEFINES += USELIBSWRESAMPLE
-            LIBS    += -lswresample                                             #------ conditionnaly include libswresample
-        } else:exists(/usr/include/libavresample/avresample.h) {
-            message("Use libav 9 in /usr/include")
-            DEFINES += USELIBAVRESAMPLE
-            LIBS    += -lavresample                                             #------ conditionnaly include libavresample
-            INCLUDEPATH += /usr/include/
-        } else {
-            message("Use libav 0.8+taglib in /usr/include")
-            LIBS        += -ltag                                                #------ TAGlib is used only with LIBAV 8
-            DEFINES     += USETAGLIB
-            DEFINES     += HAVE_CONFIG_H                                        #------ specific for TAGLib
-            DEFINES     += TAGLIB_STATIC                                        #------ specific for TAGLib
-            INCLUDEPATH += /usr/include/
-        }
+        message("Use ffmpeg")
+        DEFINES += USELIBSWRESAMPLE
+        LIBS    += -lswresample
 
     }
 
